chart/tests/test_basic_helm_chart.py

```diff
@@ -359,3 +359,8 @@ def test_invalid_dags_access_mode(self):
             'accessMode must be one of the following: "ReadWriteOnce", "ReadOnlyMany", "ReadWriteMany"'
             in ex_ctx.exception.stderr.decode()
         )
+
+    @parameterized.expand(["abc", "123", "123abc"])
```

Comment:
Related to https://github.com/astronomer/issues/issues/3813

```suggestion
    @parameterized.expand(["abc", "123", "123abc", "123-abc"])
```